README.md

```diff
@@ -6,20 +6,23 @@ Public scripts to launch near devnet, betanet and testnet node
 ```
 curl --proto '=https' --tlsv1.2 -sSfL https://up.near.dev | python3
 ```
-Nearup automatically add itself to PATH. And on each run, it self updates to latest version.
+Nearup automatically add itself to PATH: restart the terminal, or issue the command `source ~/.profile`.
+On each run, nearup self-updates to the latest version.
 
 ## Start devnet, betanet, testnet
 ```
 nearup devnet
 nearup betanet
 nearup testnet
 ```
+Where `devnet` is the nightly release; `betanet` is the weekly release; `testnet` is the stable release.
 
 ## Start devnet, betanet, testnet with officially compiled binary
 Currently Linux only:
 ```
-nearup {devnet, betanet, testnet} --nodocker
+nearup betanet --nodocker
 ```
+Replace `betanet` with `devnet` or `testnet` if you want to use a different network
 
 ## Start devnet, betanet, testnet with local nearcore
 ```
```

Comment:
Please, move the "compile in nearcore/" comment from the code block to be a simple text.